docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  ViewController.swift
+//  api-walkthrough
+//
+//  Created by James Nocentini on 27/06/2017.
+//  Copyright © 2017 couchbase. All rights reserved.
+//
+
+import UIKit
+import CouchbaseLiteSwift
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let database: Database
+        do {
+            database = try Database(name: "my-database")
+        } catch let error as NSError {
+            NSLog("Cannot open the database: %@", error)
+            return
+        }
+        
+        // create document
+        let dict: [String: Any] = ["type": "task",
+                                   "owner": "todo",
+                                   "createdAt": Date()]
+        let newTask = Document(dictionary: dict)
+        try? database.save(newTask)
+        
+        // mutate document
+        newTask.set("Apples", forKey:"name")
+        try? database.save(newTask)
+        
+        // typed accessors
+        newTask.set(Date(), forKey: "createdAt")
+        let date = newTask.date(forKey: "createdAt")
+        
+        // database transaction
+        do {
+            try database.inBatch {
+                for i in 0...10 {
+                    let doc = Document()
+                    doc.set("user", forKey: "type")
+                    doc.set("user \(i)", forKey: "name")
+                    try database.save(doc)
+                    print("saved user document \(doc.string(forKey: "name"))")
+                }
+            }
+        } catch let error {
+            print(error.localizedDescription)
+        }
+        
+        // blob
+        let appleImage = UIImage(named: "avatar.jpg")!
+        let imageData = UIImageJPEGRepresentation(appleImage, 1)!
+        
+        let blob = Blob(contentType: "image/jpg", data: imageData)
+        newTask.set(blob, forKey: "avatar")
+        try? database.save(newTask)
+        
+        if let taskBlob = newTask.blob(forKey: "image") {
+            UIImage(data: taskBlob.content!)
+        }
+        
+        // query
+        let query = Query
+            .select()
+            .from(DataSource.database(database))
+            .where(
+                Expression.property("type").equalTo("user")
+                    .and(Expression.property("admin").equalTo(false))
+        )
+        
+        do {
+            let rows = try query.run()
+            for row in rows {
+                print("doc ID :: \(row.documentID)")
```

Comment:
In DB13, query.run() returns ResultSet which is an iterator of Result. The Result doesn't have documentID property anymore. I'm wondering if the code is compiled?

To get the document ID, the projection is needed. See [here](https://github.com/couchbase/couchbase-lite-ios/blob/feature/2.0/Swift/Tests/QueryTest.swift#L485-L514) as an example to get the document ID.